test/plugin/base_test.rb

```diff
@@ -1302,6 +1302,16 @@ def get_number(_field)
     _undefined
   end
 
+  # Get the value of a null field.
+  def get_null(_field)
```

Comment:
You shouldn't need to extract the null value. Instead, why not construct the value to be compared with and return that as `null_value`? So, here it would be just `nil`, and in the gRPC path it'd be the hash `{'nullValue'=>'NULL_VALUE'}`. Would that work?